docs/reference/cli.md

```diff
@@ -9,16 +9,18 @@ redirect_from:
   - ./commands.html
 ---
 
-# Commands (CLI) Reference
+# `lakectl` CLI Reference
 {:.no_toc}
 
-{% include toc.html %}
+`lakectl` is a command-line tool to explore manage and work with lakeFS.
```

Comment:
```suggestion
lakectl is the lakeFS command-line tool.
```

Reply:
I think this is punchier but now confusing: it's the CLI client, but we also run lakefs sometimes.

Reply:
Fair enough. How about:
```suggestion
`lakectl` is a command-line tool to communicate with the lakeFS server.
```